### PR TITLE
Further Adjustments to Starting Ages

### PR DESCRIPTION
**Pull request by IllianiCBT**
Fixed some edge cases and better refined Clan starting ages.

### CODE REVIEW

File: MekHQ/src/mekhq/Utilities.java [hunk 1]
```diff
@@ -345,10 +348,10 @@ public static int generateExpLevel(int bonus) {
 
         return switch (roll) {
             case 1 -> SkillType.EXP_ULTRA_GREEN;
```
Comment (Sleet01) [inline MekHQ/src/mekhq/Utilities.java:350]: Can we remove `SkillType.` here as well?

Comment (IllianiCBT) [reply MekHQ/src/mekhq/Utilities.java:350]: Yes we certainly can - I don't know why that didn't get nixed automatically

File: MekHQ/src/mekhq/Utilities.java [hunk 2]
```diff
@@ -891,13 +915,18 @@ public static int getAgeByExpLevel(int experienceLevel, boolean isClan) {
...
 
-            if (isClan) {
+            if (isClan && (experienceLevel != EXP_NONE)) {
                 roll = (int) Math.ceil(roll / 2.0);
             }
 
             age += roll;
         }
 
+        // Clam age, if necessary
```
Comment (Sleet01) [inline MekHQ/src/mekhq/Utilities.java:925]: :clamp: rather than :oyster: I think.

**Review by Sleet01 — CHANGES_REQUESTED** [review]
Just a couple touch-ups, then good to go.

**Review by Sleet01 — APPROVED** [review]

**Pull request merged**